kikit/panelize_ui_impl.py

```diff
@@ -253,18 +256,31 @@ def buildLayout(preset, panel, sourceBoard, sourceArea):
                 vbonefirst=layout["vbonefirst"])
             substrates = panel.makeGrid(
                 boardfile=sourceBoard, sourceArea=sourceArea,
-                rows=layout["rows"], cols=layout["cols"], destination=VECTOR2I(0, 0),
+                rows=layout["rows"], cols=layout["cols"], destination=layout["destination"],
```

Comment:
What is the point of destination? Any changes made to the destination will be overwritten by page properties `posx` and `posy`. See https://yaqwsx.github.io/KiKit/v1.3/panelization/cli/#page